tests/stubdata/output/jats_springer_JHEP_JHEP07_2023_200.json

```diff
@@ -11,6 +11,26 @@
         "collab": "The ALICE collaboration"
       }
     },
+    {
```

Comment:
This new change is because the collab is now treated as an "author" so all the affIDs are being parsed with one of the recent changes we made